.circleci/config.yml

```diff
@@ -12,12 +12,12 @@ jobs:
           keys:
             - v2-cache-source-{{ .Branch }}-{{ .Revision }}
             - v2-cache-source-{{ .Branch }}
+      - checkout
       - run:
           name: Initialize submodules
           command: |
             git submodule init
             git submodule update
-      - checkout
```

Comment:
This isn't directly related to the PR, but was blocking this from building. These need to be swapped b/c `git submodule init` will only work if there is a git repo already setup. This is only the case if CircleCI is able to load up one from the cache. Swapping these makes it always work b/c the repo will be checked out if it's not loaded from cache.